WTWSMS/events/WtWSMS_migration_events.txt

```diff
@@ -236,14 +236,6 @@ character_event = {
 	
 	trigger = {
 		NOT = { year = 800 } # The migration period ended around 800
-		OR = { # Possible target religion groups
-			religion_group = christian
-			religion = finnish_pagan
-			religion = germanic_pagan
-			religion = baltic_pagan
-			religion = scythian_pagan
-			religion = slavic_pagan
-		}
```

Comment:
Okay. I won't touch the mechanic without explicit permission from here on out.

-but you SERIOUSLY need to implement a Rule to allow for random Migration. Watching Norsemen show up in Turkestan is exciting and dynamic to me. You are right to say it is ahistorical, but you shouldn't shut it down because of it. The Migration mechanic is the biggest feature of this Mod and we should have the choice on how crazy it gets.